src/app/organizations/state/register-organization.service.ts

```diff
@@ -198,4 +210,30 @@ export class RegisterOrganizationService {
       });
     }
   }
+
+  gravatarUrl(imgLink: string): string {
```

Comment:
Are you saving the gravatar url in the API? I think you should save the simple url, then prepend it with the gravatar url when displaying it.

Reply:
No, the url someone enters through the api is what gets saved. Not the gravatar version.

Reply:
What does line 192 do? It looks like it's calling this method which will transform the url.

Reply:
yes, it makes the url into a gravatar one before saving it into the database.

Reply:
I think you should save it without adding the gravatar portion. You already need to handle the case where there's no Gravatar url, if somebody already entered one directly via the API, so I think it makes your life simpler to just store "plain" urls.

Reply:
Yes, it's much simpler now doing that.